grading-assigner.py

```diff
@@ -84,7 +84,7 @@ def fetch_certified_pairs():
     logger.info("Requesting certifications...")
     me_resp = requests.get(ME_URL, headers=headers)
     me_resp.raise_for_status()
-    languages = me_resp.json()['application']['languages'] or ['en-us']
+    languages = me_resp.json()['mentor_languages'] or ['en-us']
```

Comment:
Shouldn't it be `or ['en']` ?

Reply:
@ivanlares The original script defaulted it to `en-us`. Do you think the correct default is `en`?

Reply:
@Deborah-Digges My mistake, when I printed the json thought I saw 'en' instead of 'en-us'. 
I just double checked the documentation and it does use 'en-us'.